sdcm/stress_thread.py

```diff
@@ -77,6 +78,7 @@ def __init__(self, loader_set, stress_cmd, timeout, output_dir, stress_num=1, ke
 
         self.executor = None
         self.results_futures = []
+        self.shell_marker = ''.join(random.choice(string.ascii_uppercase + string.digits) for x in range(20))
```

Comment:
1) add a comment explaining the need of the shell marker
2) create a utils function that creates random string of length N